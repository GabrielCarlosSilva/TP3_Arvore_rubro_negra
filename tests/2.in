1
4
Lucas
22
Luisa
21
Beatriz 
32
Bruna 
25
2
1
3
Lais
20
Maria
54
Lorena
19
0
