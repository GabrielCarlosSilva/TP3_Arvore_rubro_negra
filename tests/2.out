Dados inOrder:
Nome: Luisa
Idade: 21
Nome: Lucas
Idade: 22
Nome: Bruna
Idade: 25
Nome: Beatriz
Idade: 32
Dados inOrder:
Nome: Lorena
Idade: 19
Nome: Lais
Idade: 20
Nome: Luisa
Idade: 21
Nome: Lucas
Idade: 22
Nome: Bruna
Idade: 25
Nome: Beatriz
Idade: 32
Nome: Maria
Idade: 54